src/WebManager/Azure.ContainerRegistry/Docker/MediaTypes.cs

```diff
@@ -0,0 +1,8 @@
+namespace Azure.ContainerRegistry.Docker
```

Comment:
Could this be the following instead?
```suggestion
namespace WebManager.Utility.Docker
```